drive/amazon/amazon.go

```diff
@@ -46,29 +47,30 @@ func NewClient(c drive.Config) (drive.Client, error) {
 	if err != nil {
 		return nil, err
 	}
-	return &AmazonCloudDrive{
+	return &Drive{
```

Comment:
I'm okay with dropping Amazon, to avoid the stutter, but please keep Cloud in CloudDrive, as it's technically part of the product's name:
https://www.amazon.com/clouddrive